docker/compose/aptos-node/docker-compose.yaml

```diff
@@ -0,0 +1,101 @@
+# This compose file defines an Aptos node deployment.
+# It includes a validator node and a fullnode.
+# To use this docker compose:
+# - Generate keys: <TODO add commands here>
+# - Download genesis.blob and waypoint.txt
+# - Initialize secure-data.json
```

Comment:
We'll need tooling for this, too, right? Unless we start all at a predefined state, e.g., 0s?

Reply:
Yes we need the tooling for this too, should add a <TODO> sign here.